tests/PHPStan/Rules/Functions/FunctionCallableArgExistsRuleTest.php

```diff
@@ -0,0 +1,211 @@
+<?php declare(strict_types = 1);
+
+namespace PHPStan\Rules\Functions;
+
+use PHPStan\Rules\CallableExistsCheck;
+
+class FunctionCallableArgExistsRuleTest extends \PHPStan\Testing\RuleTestCase
+{
+
+	protected function getRule(): \PHPStan\Rules\Rule
+	{
+		$broker = $this->createBroker();
+		return new FunctionCallableArgExistsRule(
+			$broker,
+			new CallableExistsCheck($broker)
+		);
+	}
+
+	public function testCallableExists()
+	{
+		$lineOffset = 39;
+		$lineOffset2 = 74;
+
+		$this->analyse([__DIR__ . '/../data/callable-exists.php'], [
+					[
+				'Argument #1 of call_user_func should be callable, but passed array has too much items, only two items expected.',
+				$lineOffset, //109
```

Comment:
Please don't use this, write the literal line where the error occurs.

Reply:
I will change it in final version, I often add new test cases whoch changes line numbers so i create offset to fix tests more easily.